### DISCUSSION TITLE
Show what has changed since the last backup, before actually backing it up (--dry-run?)

### DISCUSSION THREAD

**Discussion by christfried** [borgbase/vorta#1282] [FAQ]
Is there a way to accomplish what I asked for in the header? I just tried a --dry-run but it did (almost) nothing, except for telling me where the backup would go and what the time stamp would be. When I started the real backup process afterwards it had 2.8 GB to backup, which took 0:02:12.

**Comment by m3nu** [comment]
We can support what Borg supports. If they provide a list of changes, it would be feasible.

**Reply by real-yfprojects** [reply]
Borg dry-run provides a list of changes/added/removed files but doesn't show the type of change or any other information.
A way to browse a preview of the files backed up also makes sense in the context of exclude patterns.

**Reply by christfried** [reply]
Right, but --dry-run did not show me anything useful here. Why is that? (This is given as an example under Schedule -> Shell Commands)

**Reply by real-yfprojects** [reply]
> Right, but --dry-run did not show me anything useful here. Why is that? (This is given as an example under Schedule -> Shell Commands)

There is no support for this option implemented into _vorta_. You will have to use the command line.
